### PR TITLE
Create Claim

### PR DESCRIPTION
**Pull request by artemfastdev**
Add ability to create a new claim for IX member; add material-ui core; add new modal window

### CODE REVIEW

File: src/components/shared/modals/MaterialModal.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+interface MaterialModalProps {
+  classes: any
+  open: boolean
+  handleClose: () => void
+  children?: ReactNode
+}
+
+const MeterialModal: React.SFC<MaterialModalProps> = (props) => {
```
Comment (hedvigoscar) [inline src/components/shared/modals/MaterialModal.tsx]:
```suggestion
const MaterialModal: React.SFC<MaterialModalProps> = (props) => {
```

File: src/components/shared/modals/MaterialModal.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  return (
+    <Modal open={props.open} onClose={props.handleClose}>
+      <div style={getModalStyle()} className={props.classes.paper}>
+        {props.children}
+      </div>
+    </Modal>
+  )
+}
+
+export default withStyles(styles)(MeterialModal)
```
Comment (hedvigoscar) [inline src/components/shared/modals/MaterialModal.tsx]:
```suggestion
export default withStyles(styles)(MaterialModal)
```

File: src/components/shared/modals/MaterialModal.tsx (hunk 1 above)
Comment (artemfastdev) [reply src/components/shared/modals/MaterialModal.tsx]: done)

File: src/components/shared/modals/MaterialModal.tsx (hunk 2 above)
Comment (artemfastdev) [reply src/components/shared/modals/MaterialModal.tsx]: done)

**Pull request merged**